plugin/storage/grpc/factory.go

```diff
@@ -38,9 +38,10 @@ type Factory struct {
 
 	builder config.PluginBuilder
 
-	store        shared.StoragePlugin
-	archiveStore shared.ArchiveStoragePlugin
-	capabilities shared.PluginCapabilities
+	store               shared.StoragePlugin
+	archiveStore        shared.ArchiveStoragePlugin
+	streamingSpanWriter shared.StreamingSpanWriterPlugin
+	capabilities        shared.PluginCapabilities
```

Comment:
I just realized something - we probably didn't need to introduce Capabilities in the code at all if we used gRPC reflection instead. But that ship has sailed.

Reply:
yeah